Shared/Samples/Display content of utility network container/DisplayContentOfUtilityNetworkContainerView.swift

```diff
@@ -0,0 +1,159 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct DisplayContentOfUtilityNetworkContainerView: View {
+    /// The display scale of this environment.
+    @Environment(\.displayScale) private var displayScale
+    
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// A Boolean value indicating whether the map view interaction is enabled.
+    @State private var isMapViewUserInteractionEnabled = true
+    
+    /// A Boolean value indicating whether the legends are shown.
+    @State private var isShowingLegend = false
+    
+    /// A Boolean value indicating whether the content within the container is shown.
+    @State private var isShowingContainer = false
+    
+    /// The map point where the map was tapped.
+    @State private var mapPoint: Point?
+    
+    /// The point to identify a graphic.
+    @State private var screenPoint: CGPoint?
+    
+    /// The viewpoint before the map view zooms into the container's extent.
+    @State private var previousViewpoint: Viewpoint?
+    
+    /// The current viewpoint of the map view.
+    @State private var viewpoint = Viewpoint(latitude: 41.80, longitude: -88.16, scale: 4e3)
+    
+    var body: some View {
+        MapViewReader { proxy in
+            MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+                .interactionModes(isMapViewUserInteractionEnabled ? .all : [])
+                .onViewpointChanged(kind: .boundingGeometry) { viewpoint = $0 }
+                .onSingleTapGesture { screenPoint, mapPoint in
+                    self.screenPoint = screenPoint
+                    self.mapPoint = mapPoint
+                }
+                .overlay(alignment: .top) {
+                    Text(model.statusMessage)
+                        .frame(maxWidth: .infinity)
+                        .padding(.vertical, 6)
```

Comment:
It may not make a different, but conceptually, you would want to apply padding first, and then adjust the frame of the padded text.

```suggestion
                    Text(model.statusMessage)
                        .padding(.vertical, 6)
                        .frame(maxWidth: .infinity)
```